certified-connectors/COSMO Bot/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1429 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "COSMO Bot",
+    "description": "COSMO Bot increases your business success through more efficiency in everyday life. If you answer a standard question today, you are wasting time. In contact with your customers, this is often a good opportunity to get to know each other and establish a relationship. But internally, from employee to employee, or when there is simply too much to do, it is a luxury that not everyone can allow themselves. This Connector allows you to extend the bots functionality and integrate other systems.",
+    "contact": {
+      "name": "COSMO CONSULT Support",
+      "url": "https://de.cosmoconsult.com/service/support/",
+      "email": "support.west@cosmoconsult.com"
+    }
+  },
+  "host": "cc-bot-master-server.azurewebsites.net",
+  "basePath": "/api/bot/v1",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://cosma.link/cc-bot"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.cosmoconsult.com/data-protection/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI;Bot;Chatbot"
```

Comment:
@jenkoc `Chatbot` is not the list of pre-defined categories. Can you please refer the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#product-or-end-service-metadata) and remove this category or add a new one from the list.

Reply:
For some additional clarity, `Bot` isn't on the list of pre-defined categories as well. We do have a current limit of two categories per connector, so `Bot` and `Chatbot` can be removed in favor of one of the other pre-defined options, or they can be removed and `AI` is standalone.

Reply:
@sriyen-msft @dougelliott Thank's for your feedback, i've updated the definition.